public/css/main.css

```diff
@@ -131,14 +135,41 @@ select {
 
 button {
 	background-color: #a74e94;
+	transform: translate3d(0, 0, 0);
```

Comment:
caniusethis.com thinks translate3d should work in this context on ie10/11 but not ie9. The audit isn't done yet, but this should be fine :-/